lib/bindings/Cargo.toml

```diff
@@ -17,7 +17,8 @@ breez-sdk-liquid = { path = "../core" }
 log = { workspace = true }
 uniffi = { workspace = true, features = [ "bindgen-tests", "cli" ] }
 # Bindgen used by KMP, version has to match the one supported by KMP
-uniffi_bindgen = "0.25.2"
+uniffi_bindgen = { workspace = true }
+uniffi_bindgen_kmp = { version = "0.25.2", package = "uniffi_bindgen" }
```

Comment:
With the new `uniffi_bindgen_kmp`, is the `uniffi_bindgen_kotlin_multiplatform` below still necessary?

Reply:
I had to split the use of `uniffi_bindgen` because of compatibility issues, so `uniffi_bindgen_kmp` is `uniffi_bindgen` 0.25.2 for use with `uniffi_bindgen_kotlin_multiplatform`. 

I will play around a bit more to see if I can get them to use the same `uniffi_bindgen`.